src/lib/lib.jl

```diff
@@ -272,13 +289,16 @@ end
   end
 end
 
-@generated function (back::Jnew{T,G,true})(Δ::Union{NamedTuple,Nothing,RefValue}) where {T,G}
-  !T.mutable && Δ == Nothing && return :nothing
+@generated function (back::Jnew{T,G,true})(Δ::Union{NamedTuple,Nothing,AbstractZero,RefValue}) where {T,G}
+  if !T.mutable
+      Δ == Nothing && return :(Zero())
+      Δ <: AbstractZero && return :(Zero())
```

Comment:
```suggestion
      Δ <: AbstractZero && return :Δ
```

so that `Zero` and `DoesNotExist` say that same